### PR TITLE
Godot 3.6 beta 5 blog post

### PR DESCRIPTION
**Pull request by lawnjelly**
This is draft ideas for @akien-mga to finish for the blog post.

### CODE REVIEW

File: collections/_article/dev-snapshot-godot-3-6-beta-5.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+This beta represents feature freeze for 3.6. We will now concentrate on bug fixing until we reach stable. Any new features will be scheduled for 3.7.
+
+[Jump to the **Downloads** section](#downloads), and give it a spin right now, or continue reading to learn more about improvements in this release. You can also [try the **Web editor**](https://editor.godotengine.org/releases/3.6.beta5/) or the **Android editor** for this release. If you are interested in the latter, please request to join [our testing group](https://groups.google.com/g/godot-testers) to get access to pre-release builds.
+
+
+## Highlights
+
+Although [beta 1](https://godotengine.org/article/dev-snapshot-godot-3-6-beta-1/), [beta 2](https://godotengine.org/article/dev-snapshot-godot-3-6-beta-2/), [beta 3](https://godotengine.org/article/dev-snapshot-godot-3-6-beta-3/) and [beta 4](https://godotengine.org/article/dev-snapshot-godot-3-6-beta-4/) contained many 2D features, beta 5 adds a number of 3D features so there should be something for everyone:
```
Comment (AThousandShips) [inline collections/_article/dev-snapshot-godot-3-6-beta-5.md]:
```suggestion
Although [beta 1](https://godotengine.org/article/dev-snapshot-godot-3-6-beta-1/), [beta 2](https://godotengine.org/article/dev-snapshot-godot-3-6-beta-2/), [beta 3](https://godotengine.org/article/dev-snapshot-godot-3-6-beta-3/), and [beta 4](https://godotengine.org/article/dev-snapshot-godot-3-6-beta-4/) contained many 2D features, beta 5 adds a number of 3D features so there should be something for everyone:
```

File: collections/_article/dev-snapshot-godot-3-6-beta-5.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+### Mesh Merging
+([GH-61568](https://github.com/godotengine/godot/pull/61568))
+
+https://docs.godotengine.org/en/3.6/tutorials/3d/merge_groups.html
+
+![MergeGroup example](/storage/blog/godot-3.6/merge_group_house.webp)
+
+Godot 3.6 now offers a comprehensive system for mesh merging, both at design time and at runtime. OpenGL can be severely bottlenecked by drawcalls and state changes when drawing lots of objects. Now you can blast through these barriers and potentially render any number of similar objects in a single drawcall.
+
+As well as allowing you to optimize existing maps and moving objects, this also makes new procedural game types possible, as thousands of procedurally placed objects can be merged at runtime so as to render efficiently (think vegetation, rocks, furniture, houses etc).
```
Comment (AThousandShips) [inline collections/_article/dev-snapshot-godot-3-6-beta-5.md]:
```suggestion
As well as allowing you to optimize existing maps and moving objects, this also makes new procedural game types possible, as thousands of procedurally placed objects can be merged at runtime so as to render efficiently (think vegetation, rocks, furniture, houses, etc.).
```

File: collections/_article/dev-snapshot-godot-3-6-beta-5.md [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+
+### ORM Materials
+([GH-76023](https://github.com/godotengine/godot/pull/76023))
+
+Ansraer adds support for ORM materials, which is a standard format where occlusion, roughness and metallic are combined into a single texture. This means these standard PBR textures can be used without modification, and rendering performance will likely be increased where they are used (compared to the old workflow).
```
Comment (AThousandShips) [inline collections/_article/dev-snapshot-godot-3-6-beta-5.md]:
```suggestion
Ansraer adds support for ORM materials, which is a standard format where occlusion, roughness, and metallic are combined into a single texture. This means these standard PBR textures can be used without modification, and rendering performance will likely be increased where they are used (compared to the old workflow).
```

File: collections/_article/dev-snapshot-godot-3-6-beta-5.md [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+Fixes to physics interpolation, and hierarchical culling, as well as performance increases.
+
+## Editor
+
+### View Selected Mesh Stats
+([GH-88207](https://github.com/godotengine/godot/pull/88207))
+
+![View mesh stats example](/storage/blog/godot-3.6/view_mesh_stats.webp)
+
+The 3D view menu now offers a new (long overdue) option, "view selected mesh stats". This will display total triangle counts, vertex counts and index counts for the selected meshes (and multimeshes).
```
Comment (AThousandShips) [inline collections/_article/dev-snapshot-godot-3-6-beta-5.md]:
```suggestion
The 3D view menu now offers a new (long overdue) option, "view selected mesh stats". This will display total triangle, vertex, and index counts for the selected meshes (and multimeshes).
```
Or just:
```suggestion
The 3D view menu now offers a new (long overdue) option, "view selected mesh stats". This will display total triangle counts, vertex counts, and index counts for the selected meshes (and multimeshes).
```

**Comment by akien-mga** [comment]
TL;DR for socials (@MurderVeggie):

- Finally a new beta for 3.6, a year after the first beta
- Lots of 3D performance optimizations, such as discrete Level of Detail and mesh merging
- 3.6 now feature complete, focusing on bug fixes to get to stable release

Illustration: Backpack Battles by PlayWithFurcifer
https://store.steampowered.com/app/2427700/Backpack_Battles/

backpack_battles2

URL: https://godotengine.org/article/dev-snapshot-godot-3-6-beta-5/

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!
